src/node_zlib.cc

```diff
@@ -607,9 +607,11 @@ class CompressionStream : public AsyncWrap, public ThreadPoolWork {
     return AllocForBrotli(data, real_size);
   }
 
+  static constexpr size_t reserveSizeAndAlign =
+      std::max(sizeof(size_t), alignof(max_align_t));
```

Comment:
Isn't this just always `alignof(max_align_t)` anyway?

Reply:
no, it you don't care about unaligned accesses you could set it to 1. There is no architecture actually doing that though. But this just addressing a comment made by @Flarna.

Reply:
see https://github.com/nodejs/node/pull/57727#issuecomment-2775059845 as well, this just remove a double definition of a constant, there is nothing special about this.

Reply:
I'm not aware of any platform where `alignof(max_align_t)` is smaller then `sizeof(size_t)` as most hardware prefers aligned data.
But to my understanding C++ standard allows `alignof(max_align_t) == 1` but the space needed here is `sizeof(size_t)` so take the max.

Reply:
@addaleax Just make a large struct or class and you can easily check that the alignment of it is less then its size. This code is simply doing the right thing, it will make sure there is enough space and it is still aligned. In theory alignof(max_align_t) can be less than sizeof(size_t). In practice is won't, but the compiler will optimize it out anyway.